File: python/program/Civitas.py
from Human import *

class Civitas(Human):
    '''
    Kelas Civitas terdiri dari
    atribut2 Human dan juga
    univerity dan email.
    Terdiri dari metode2
    setter dan getter
    '''
    
    def __init__(self, name, nik, gender, university, email):
        super().__init__(name, nik, gender)
        self._university = university
        self._email = email
    
    def setUniversity(self, university):
        self._university = university
    
    def setEmail(self, email):
        self._email = email
    
    def getUniversity(self):
        return self._university
    
    def getEmail(self):
        return self._email

File: java/program/Civitas.java
public class Civitas extends Mahasiswa {

    protected String university;
    protected String email;
    
    Civitas(String name, String nim, Prodi prodi, String fakultas, String nik, String gender, String university, String email){
        super(name, nim, prodi, fakultas, nik, gender);
        this.university = university;
        this.email = email;
    }

    public void setUniversity(String university) {
        this.university = university;
    }
    public void setEmail(String email) {
        this.email = email;
    }

    public String getUniversity() {
        return university;
    }
    public String getEmail() {
        return email;
    }

}
